TYPO3.Flow/Classes/TYPO3/Flow/Http/Headers.php

```diff
@@ -450,8 +465,16 @@ protected function setCookiesFromRawHeader($rawFieldValue)
                 continue;
             }
             list($name, $value) = explode('=', $cookiePair, 2);
-            if (trim($name) !== '' && preg_match(Cookie::PATTERN_TOKEN, $name) === 1) {
-                $this->setCookie(new Cookie(trim($name), urldecode(trim($value, "\t ;\""))));
+            $trimmedName = trim($name);
+
+            if ($trimmedName === '') {
+                continue;
+            }
+
+            if (preg_match(Cookie::PATTERN_TOKEN, $trimmedName) !== 1) {
+                $this->systemLogger->log(sprintf('Cookie name %s is not RFC 2616 compliant, therefore it will be skipped', $trimmedName), LOG_INFO);
```

Comment:
question is if it should throw an exception in development context, but ignore in production or just always ignore..